libs/sdk-core/src/models.rs

```diff
@@ -1114,6 +1117,18 @@ pub enum LnUrlCallbackStatus {
     },
 }
 
+/// [LnUrlCallbackStatus] specific to LNURL-withdraw, where the success case contains the invoice.
+#[derive(Serialize)]
+pub enum LnUrlWithdrawCallbackStatus {
```

Comment:
Perhaps if we are introducing new types here we can use better naming?
`LnUrlWithdrawCallbackStatus`  => `LnUrlWithdrawResult`
`LnUrlWithdrawOkData`  => `LnurlWithdrawSuccess`
`LnUrlErrorData`  => `LnurlError`